runtime/containerd/containerd.go

```diff
@@ -39,14 +39,14 @@ const (
 func init() {
 	runtime.Register(dockerRuntimeName, func() runtime.ContainerRuntime {
 		return &ContainerdRuntime{
-			Mgmt: new(types.MgmtNet),
-		}
+			Mgmt: new(types.MgmtNet)}
 	})
 }
 
 func (c *ContainerdRuntime) Init(opts ...runtime.RuntimeOption) error {
 	var err error
 	log.Debug("Runtime: containerd")
+	c.keepMgmtNet = false
```

Comment:
no need to set it, this is the default value

Reply:
ok removed it